### DISCUSSION TITLE
plastic scintillator compatibility

### DISCUSSION THREAD

**Discussion by larizkaf** [OpenGammaProject/Open-Gamma-Detector#65] [Q&A] [answered]
Hello, I have previously purchased your open gamma detector and would like to improve it using my manufactured samples and a commercial plastic scintillator. Does using a plastic scintillator make sense? My objective is to use PMT and SiPM to compare the pulse height spectra of plastic scintillators. I also intend to assess the spectra using your MCA. Does it function?

**Comment by NuclearPhoenixx** [comment, marked as answer]
Hi there, I've tried to discuss this in my reference document: https://github.com/OpenGammaProject/Open-Gamma-Detector/blob/main/REFERENCE.md#can-i-use-a-plastic-scintillator

In general, plastic scintillators do not make sense for this project except in some very special cases. If you only want to have a cps reading and nothing else, you can use it, though there are other much simpler projects that can do the same. I have not yet used a plastic scintillator successfully, although I have not really invested any time in testing them. I'm pretty sure they don't work out of the box since the stock gain is too low. Please correct me if I'm wrong!

**Reply by larizkaf** [reply]
Hi, I appreciate your reply. So, is there a change in the coding software or can I use it directly if all I want is the CPS reading? Another query: is it feasible for me to connect your open gamma detector board, especially for SiPM biasing voltage and your SiPM board to the amplifier or digitizer?

**Reply by NuclearPhoenixx** [reply]
No, you can use it directly there is no change to the software needed.

Regarding your second question, yes that's possible if I understand correctly. Just be sure to use a common ground. :)

**Reply by saihariharan15** [reply]
If you are open to testing plastic scintillators, consider the following steps:

    Enhance Gain: Modify the amplification system to address the low stock gain issue.
    Compare with Other Detectors: Test them alongside more standard scintillators (e.g., NaI(Tl) or CsI(Tl)) to gauge relative performance.
    Document Findings: Share the results of any experiments to contribute to the broader knowledge base.

Conclusion

Your assessment is likely correct that plastic scintillators don't work effectively "out of the box" for this project. Unless a specific use case justifies their inclusion, it seems prudent to focus on other scintillator types better suited to the project's objectives.

**Comment by saihariharan15** [comment]
If you are open to testing plastic scintillators, consider the following steps:

    Enhance Gain: Modify the amplification system to address the low stock gain issue.
    Compare with Other Detectors: Test them alongside more standard scintillators (e.g., NaI(Tl) or CsI(Tl)) to gauge relative performance.
    Document Findings: Share the results of any experiments to contribute to the broader knowledge base.

Conclusion

Your assessment is likely correct that plastic scintillators don't work effectively "out of the box" for this project. Unless a specific use case justifies their inclusion, it seems prudent to focus on other scintillator types better suited to the project's objectives.

**Reply by NuclearPhoenixx** [reply]
I agree with your conclusion 👍🏻

**Comment by larizkaf** [comment]
Hi I already tried to connect SiPM board and your electronic board connect to ORTEC 572A amplifier. Then I used Cs-137 as the source. Here I attach the result. Do you think is that works?
![WhatsApp Image 2024-12-04 at 2 27 15 PM](https://github.com/user-attachments/assets/7bc79cca-bafc-43ea-be46-49cfedd94556)
![WhatsApp Image 2024-12-04 at 2 26 05 PM](https://github.com/user-attachments/assets/5f714ea5-973d-4f57-be58-941510493f31)

**Reply by NuclearPhoenixx** [reply]
Might be a bit too low of a gain, not sure. What kind of count rates did you have before (at background) and after putting the Cs-137 source next to the scintillator?

**Comment by larizkaf** [comment]
Hi sorry for the late response. I tried to enhanced the gain based on your suggestion then I got the spectra as attachment. What do you think?

Regarding your previous statement, you said that this project can be just for the cps reading. Could you explain more detail why this project just can be use as plastic scintillation counter even the stock gain is low? 
![20241219_135517](https://github.com/user-attachments/assets/067be048-b1f0-4dcc-9951-cb18eef6d706)

**Reply by NuclearPhoenixx** [reply]
That looks a lot better. Plastic scintillator intrinsically have next to no usable energy resolution, so you will end up mostly only collecting cps and a very very rough energy spectrum that will not allow you to see any specific isotopes.

**Comment by larizkaf** [comment]
Okay. Thank you so much. Have a great day! :)

**Comment by larizkaf** [comment]
Hi, sorry to bother you again. I'm trying to couple the plastic scintillator to your board, as you mentioned it's possible for just counting. However, when I took CPS data using Cs-137, the CPS is always 0, with occasional counts of 1–2 CPS (as shown in the attached video). I've adjusted the threshold, but it still shows 0 CPS most of the time. Could you please suggest what I should do to get accurate CPS measurements? this is the video link: https://drive.google.com/file/d/1i00cGhrvx_WhZRHqtYKMf_tG7NcXXTYd/view?usp=sharing

**Reply by NuclearPhoenixx** [reply]
I'm confused, I thought you already had a working setup? See https://github.com/OpenGammaProject/Open-Gamma-Detector/discussions/65#discussioncomment-11853311

**Reply by larizkaf** [reply]
No, that's different. In the spectrum above, I connect the SiPM+your SiPM board to the amplifier module (ORTEC 572A). In this recent issue, I fully used your board for counting. But it seems not to work for counting.

**Reply by NuclearPhoenixx** [reply]
Oh I see, you need to increase the quite gain a lot on the board as well. That's why you only see 1 or 2 cps occasionally: https://github.com/OpenGammaProject/Open-Gamma-Detector/blob/main/REFERENCE.md#how-do-i-change-the-gain

**Reply by larizkaf** [reply]
Should I use a specific compatible potentiometer, or are there other suitable options? Besides the gain issue, are there any other potential concerns?

**Reply by NuclearPhoenixx** [reply]
You can alter the resistors themselves as well of course. The potentiometer is just an option to have it easier to change as you go. I can't think of any other concerns right now.

**Comment by larizkaf** [comment]
Hi,
I already modified the gain, and it worked—the background was no longer at 1-2 CPS. However, when I exposed the plastic scintillator to the source (both beta and gamma), it didn’t seem to respond well except when I exposed the sample to the source (Cs-137) at 0 cm. Please check the video:  https://drive.google.com/file/d/1jHkUTpQSqZfbcLf4x7tJCWSz_Nw2WCWV/view?usp=sharing

Additionally, I connected the digital pin to the oscilloscope (file attached), but the signal wasn’t perfect. What could be the issue?
Could it be due to improper packaging causing a light leak?
Is there a mismatch between the SiPM and plastic scintillator dimensions?
Or could there be an electronic issue?
![20250218_093028](https://github.com/user-attachments/assets/8802c83f-0d66-4fcf-86cb-7853252720e2)

**Reply by NuclearPhoenixx** [reply]
Nice, ~30cps doesn't look too bad! What specific scintillator material are you using?

The geometry of the scintillator and SiPM isn't 100% ideal, but it should work anyways. What definitely does make a difference is that you should wrap all of the sides of the scintillator that aren't connected to the SiPM with some reflective material, either aluminium foil or PTFE tape or something like that. As is is now, a lot of the light from the scintillator can just escape into the case and doesn't get to the SiPM. It also helps to put some silicone grease between the SiPM and the scintillator to reduce reflections. I can't really tell if you already did that.

From what I can tell you used the 3D printed case, while 3D-printing doesn't always guarantee that the case will be 100% light-tight, I don't actually think that that's the problem since you also had the additional cardboard around it and the count rate seems plausible.

Not sure what I'm seeing in the scope image, what pin/pad are we looking at?

**Reply by larizkaf** [reply]
Hi, I used Eljen 212 PVT-based scintillators.  
The image above is the output signal from pin 18 digital raspi. I connected the oscilloscope to see the square wave. But in the picture above, it wasn't the perfect square digital signal. Besides that, the cps also didn't work when the gamma source was far away (1 cm), even though you said that the count rate seems plausible because it is very near between the source and the scintillator (0 cm).

**Reply by NuclearPhoenixx** [reply]
Very weird, what average voltage is that pin 18 on your oscilloscope? Definitely doesn't look right. It should be a square wave signal with 3.3V amplitude and duration of a few 10s of µs.

Did you try to cover the crystal?

There might also be an issue with noise in the long cables.

It's really hard to debug from afar, so sorry if I'm not very helpful.

**Reply by larizkaf** [reply]
In this case, what is the sampling rate and frequency of oscilloscope to get good square wave signal result? I used 200MHz oscilloscope before. Is the oscilloscope specification influence the signal?

**Reply by NuclearPhoenixx** [reply]
I have a 100 MHz 1GSa/s DSO and have no issues with sampling.
